bluebutton-data-pipeline-fhir-load/src/main/java/gov/hhs/cms/bluebutton/datapipeline/fhir/transform/DataTransformer.java

```diff
@@ -951,7 +974,60 @@ private TransformedBundle transformBeneficiary(RifRecordEvent<BeneficiaryRow> ri
 		beneficiary.addIdentifier().setSystem(CODING_SYSTEM_CCW_BENE_ID).setValue(record.beneficiaryId);
 		beneficiary.addAddress().setState(record.stateCode).setDistrict(record.countyCode)
 				.setPostalCode(record.postalCode);
-		// TODO rest of mapping
+		if (record.birthDate != null) {
+			beneficiary.setBirthDate(Date.valueOf(record.birthDate));
+		}
+
+		switch (record.sex) {
+		case ('M'):
+			/* Male */
+			beneficiary.setGender((AdministrativeGender.MALE));
+			break;
+		case ('F'):
+			/* Female */
+			beneficiary.setGender((AdministrativeGender.FEMALE));
+			break;
+		default:
+			/* Unknown */
+			beneficiary.setGender((AdministrativeGender.UNKNOWN));
+			break;
+
+		}
+		// TODO Could not map the following fields. Initially created extension
+		// for these fields but decided this may not be the best approach.
+		// Have created a JIRA ticket called "Finalize fields for Beneficary" to
+		// revisit on where to best map the following fields.
+		/*
+		 * beneficiary.addExtension().setUrl(CODING_SYSTEM_CCW_BENE_RACE_CD)
```

Comment:
HL7 appear unwilling to add race as a standard field, since it's largely very locale-specific (i.e. what I might count as one ethnicity, someone from another country might disagree with). FHIR does have a coding system for it in STU3 (https://www.hl7.org/fhir/v3/Ethnicity/index.html), but nothing that points to it.
It seems that the Argonaut project *is* including this concept in their DAF resources/extensions, but that doesn't currently apply to what we're doing.
On balance, I'd recommend omitting it, unless/until a beneficiary or developer requests that we include it.